update sdk to 3.3.4

lzan13 opened the pull request.

lzan13 committed:
update sdk to 3.3.4

greenmemo commented:
LGTM

githubzhaoliang commented on app/src/main/java/com/hyphenate/chatuidemo/DemoHelper.java:
```diff
@@ -563,8 +563,9 @@ private void registerMessageListener() {
-                            getNotifier().onNewMsg(message);
+                        } else {
+                            // FIXME: conflict with group push notification if it's turned off, push still come in when phone is unlocked as the app is woken up by the phone to run in the background
+//                            getNotifier().onNewMsg(message);
```
this will disable all new message notification  @jerrywu12 , maybe you can check message type here to just ignore group message.

jerrywu12 replied:
@githubzhaoliang so sounds like this is a bug that we need to solve.

jerrywu12 committed:
revert changes

Notice that this will enable background notification for extended period of time. will be fixed soon.
